third_party/accessibility/ax/platform/ax_platform_node_win.cc

```diff
@@ -5408,7 +5418,7 @@ AXPlatformNodeWin* AXPlatformNodeWin::GetTargetFromChildID(
 
   AXPlatformNodeBase* base =
       FromNativeViewAccessible(node->GetNativeViewAccessible());
-  if (base && !IsDescendant(base))
+  if (base && !base->IsDescendantOf(this))
```

Comment:
Allows using the descendant's override of `IsDescendantOf` to handle whichever delegate type it has.